configurator/mise.toml

```
@@ -0,0 +1,32 @@
+[tools]
+uv = "0.9"
+
+[tasks.deps]
+description = "Install dependency"
+run = "uv sync"
+alias = "build"
+sources = [
+    "pyproject.toml",
+    "uv.lock",
+]
+outputs = { auto = true }
+
+[tasks.lint]
+depends = ["deps"]
+description = "Run linter"
+run = ["uv run task lint", "uv run task check"]
+
+[tasks.format]
+depends = ["deps"]
+description = "Run Formatter"
+run = "uv run task format"
+
+[tasks.test]
+depends = ["deps"]
+description = "Run tests"
+run = "uv run task test"
+
+[tasks.configurator]
```

```suggestion
[tasks.start]
```

configurator/README.md 内の起動コマンドや他のシステムに合わせて`mise start`への修正

---

https://github.com/CoRE-MA-KING/roboapp/blob/5990f8e23f752af1bd87f581f50be845f3b0532c/mise.toml#L7-L11
と合わせたつもりでしたが、他がstartなのでそのほうが良さそうです
aliasをstartに貼ろうと思います